anisoap/representations/radial_basis.py

```diff
@@ -57,3 +69,142 @@ def compute_gaussian_parameters(self, r_ij, lengths, rotation_matrix):
             center -= 1 / sigma**2 * np.linalg.solve(precision, r_ij)
 
         return precision, center
+
+    def gto_square_norm(self, n, sigma):
+        """
+        Compute the square norm of GTOs (inner product of itself over R^3).
+        A GTO of order n is \phi_n = r^n * e^{-r^2/(2*\sigma^2)}
+        The square norm of the GTO has an analytic solution:
+        <\phi_n | \phi_n> = \int_0^\infty dr r^2 |\phi_n|^2 = 1/2 * \sigma^{2n+3} * \Gamma(n+3/2)
+        Args:
+            n: order of the GTO
+            sigma: width of the GTO
+
+        Returns:
+            square norm: The square norm of the GTO
+        """
+        return 0.5 * sigma**(2 * n + 3) * gamma(n + 1.5)
+
+    def gto_prefactor(self, n, sigma):
+        """
+        Computes the normalization prefactor of a GTO.
+        This prefactor is simply 1/sqrt(square_norm_area).
+        Scaling a GTO by this prefactor will ensure that the GTO has square norm equal to 1.
+        Args:
+            n: order of GTO
+            sigma: width of GTO
+
+        Returns:
+            N: normalization constant
+
+        """
+        return np.sqrt(1/self.gto_square_norm(n, sigma))
```

Comment:
These would probably make more sense in util, rather than in radial_basis.

Reply:
Or at the very least outside of the class.

Reply:
I placed these functions outside the class